src/components/trash-page/trash-page.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import styles from './trash-page.module.scss';
+import { FakeNote } from '../notes/fake-note/fake-note';
+import { useNotesStore } from '../../data-management/use-notes-store';
+import { useEffect, useMemo, useState } from 'react';
+import { SearchInput } from '../search-input/search-input';
+
+export interface TrashPageProps {
+    className?: string;
+}
+
+export const TrashPage = ({ className }: TrashPageProps) => {
+    const { notes, syncNotes } = useNotesStore();
+
+    const [searchString, setSearchString] = useState('');
+
+    useEffect(() => {
+        syncNotes();
+    }, []);
+
+    const relevantNotes = useMemo(() => {
+        return searchString
+            ? notes.filter((note) => note.deletedAt && note.title.includes(searchString))
+            : notes.filter((note) => note.deletedAt);
+    }, [notes, searchString]);
+
+    return (
+        <div className={classNames(styles.root, className)}>
+            <SearchInput
+                className={styles.searchInput}
+                value={searchString}
+                placeholder="Search deleted notes"
+                onChange={setSearchString}
+            />
+            <div className={styles.title}>Trash</div>
```

Comment:
You can use a section component here, i guess.
And you can wrap it with ScrollArea component

Reply:
I've added scroll, but I don't see the thumb. Maybe you can help me to understand why.

Reply:
We have ScrollArea component in our project. Please use it